etc/profile-m-z/tidal-hifi.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for tidal-hifi
+# Description: The web version of Tidal running in electron with hifi support thanks to widevine.
```

Comment:
So it could/should be a electron redirect?

Reply:
I'll go ahead and make it an electron redirect 👍🏽